--- pacman.conf.old	2017-07-15 00:47:31.703333326 +0000
+++ pacman.conf	2017-07-15 00:48:13.873333333 +0000
@@ -90,8 +90,12 @@ Include = /etc/pacman.d/mirrorlist
 #[multilib-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
+
+[archlinuxfr]
+SigLevel=Never
+Server=http://repo.archlinux.fr/$arch
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
